src/Targets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -15,6 +15,8 @@
     <AppDesignerFolderContentsVisibleOnlyInShowAllFiles Condition="'$(AppDesignerFolderContentsVisibleOnlyInShowAllFiles)' == ''">false</AppDesignerFolderContentsVisibleOnlyInShowAllFiles>
     <LanguageServiceName Condition="'$(LanguageServiceName)' == ''">C#</LanguageServiceName>
     <TemplateLanguage Condition="'$(TemplateLanguage)' == ''">CSharp</TemplateLanguage>
+    <!-- Shared Projects need to use the shared project item template guid. We determine shared projects by the shproj extension of the project file. -->
+    <AddItemTemplatesGuid Condition="('$(AddItemTemplatesGuid)' == '') AND ('$(ProjectExt)' == 'shproj'">{DDDDC6BE-451D-46DD-A712-A5D07027E072}</AddItemTemplatesGuid>
```

Comment:
I am looking at one of the sample outputs and I see that ProjectExt as `.shproj` rather than `shproj`.